src/Bank/BankLogic/BankAccount.cs

```diff
@@ -47,10 +56,15 @@ public bool HasEnoughCollateral(int amount)
             return amount > 0 && Balance >= amount * ratio;
         }
 
+        public int Fee()
```

Comment:
I am not seeing this method being used anywhere, why do we need it?

Reply:
This is a new requirements of the bank, to charge a yearly maintenance fee of $5 for foreign

Reply:
Ok but the logic for change yearly fee is not implemented yet right?

Reply:
Yes, I only added the relation